core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/ServerMetadataResource.java

```diff
@@ -77,4 +79,17 @@ public ServerClusterId getClusterId() {
   public SchemaRegistryServerVersion getSchemaRegistryVersion() {
     return new SchemaRegistryServerVersion(AppInfoParser.getVersion(), AppInfoParser.getCommitId());
   }
+
+  @GET
+  @Path("/type")
```

Comment:
"type" is a bit overloaded, we have another endpoint `/schemas/types`, then returns "AVRO, PROTOBUF, JSON".   I would prefer the term "deployment", but I'll leave it up to you.